mappings/net/minecraft/entity/passive/SnowgolemEntity.mapping

```diff
@@ -1,4 +1,5 @@
-CLASS arl net/minecraft/entity/passive/SnowmanEntity
+CLASS arl net/minecraft/entity/passive/SnowgolemEntity
```

Comment:
SnowGolemEntity, also SNOW_GOLEM_FLAGS, if you want to go down this path

Reply:
Isn't the entity ID snowman though?

Reply:
It's `minecraft:snow_golem`, not sure if it used to be snowman.